.github/mkosi.pkgmngr/gentoo/etc/portage/package.use/systemd

```diff
@@ -0,0 +1 @@
+sys-apps/systemd boot
```

Comment:
Can you get rid of the `boot` flag upstream when systemd v254 releases? Given that gnu-efi isn't necessary anymore from v254 onwards I'm not sure if putting sd-boot behind a flag still provides much value

Reply:
i'm not the maintainer but i see it's used in number of places [here](https://github.com/gentoo/gentoo/blob/master/sys-apps/systemd/systemd-9999.ebuild):
```
...
boot? ( $(python_gen_any_dep 'dev-python/pyelftools[${PYTHON_USEDEP}]') )
...
$(meson_native_use_bool boot bootloader)
...
```

Reply:
Yeah unless gentoo really cares about the extra python build dependencies, I would get rid of `boot` in the systemd package and just build the boot stuff by default.